recipes/demquery/meta.yaml

```
@@ -0,0 +1,46 @@
+{% set version = "0.2.0" %}
+
+package:
+  name: demquery
+  version: {{ version }}
+
+source:
+  url: https://files.pythonhosted.org/packages/28/dc/119312bc384c4a6526fdc0e4d655552516d812147c890539f92ad8924445/demquery-{{ version }}.tar.gz
+  sha256: 19650a37a5da271af5697d429db8f6cf6d43804dc46113fa622b80acd982d1f8
+  md5: e9792a03509ae3a3eec852f842e90090
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - python
```

You will need to limit ``python`` here as well, please add
```suggestion
    - python >=3.5.0
```